pysal/contrib/viz/plot.py

```diff
@@ -0,0 +1,47 @@
+"""
+Moran Plot using PySAL and Matplotlib
+
+"""
+
+__author__ = "Marynia Kolak <marynia.kolak@gmail.com>"
+
+import pandas as pd
+import numpy as np
+import pysal as ps
+import matplotlib.pyplot as plt
+
+
+def Moran(var, w, **kwargs):
```

Comment:
Two questions:

1. What'd be `**kwargs`? I don't see inside the method where those could be passed.
1. Rather than asking for the `var` and the `w` object, I think it'd be nicer to have it only take a `Moran` object. Then we can pull in all the parts we need for the plot. The rationale for that is that the user would compute a `Moran` object, check the value and, if (s)he wanted to get a visual representation would just pass that object to this method. This approach would also allow us to easily hook up this method inside `Moran` so you could (down the line) do things like:

```
m = ps.Moran(y, w)
m.plot()
```

And get the plot.